samples/cellular/nrf_cloud_multi_service/sample.yaml

```diff
@@ -94,4 +94,4 @@ tests:
     extra_args: nrf_cloud_multi_service_SHIELD=nrf7002ek
       EXTRA_CONF_FILE="overlay_nrf7002ek_wifi_no_lte.conf"
       SB_CONF_FILE="sysbuild_nrf700x-wifi-conn.conf"
-    tags: ci_build sysbuild
+    tags: ci_build sysbuild cellular
```

Comment:
This one is a Wi-Fi-only build, so you can skip it for the cellular tag

Reply:
This should probably then have `net` tag.

Reply:
Hm, that would be tricky as it is in `samples/cellular` folder. I want to keep this simple and not do cross-functional.
So in this case I prefer to remove `cellular` tag from this configuration.